tests/fsharp/core/controlMailbox/test.fsx

```diff
@@ -66,6 +66,14 @@ module MailboxProcessorBasicTests =
             100
 
 
+        check 
+            "c32398u6: MailboxProcessor null"
+            (let mb1 = new MailboxProcessor<int>(fun inbox -> async { return () })
+             mb1.StartImmediate();
```

Comment:
We need more testing than this - create a MailboxProcessor that does a side effect during it's start?